### PR TITLE
Adds 'dump contents' verb to open reagent containers, adds 'empty container into' verb to sinks and toilets and urinals

### PR DESCRIPTION
**Pull request by 9600bauds**
'dump contents': It's the exact same verb spraybottles have, but for everything
'empty container into': Like beakersplashing but doesn't make a stain if you're using one of the like five reagents in the whole game that do anything when splashed into the floor
97% tested

Adresses 2. of #8402

### CODE REVIEW

**Comment by PJB3005** [comment]
I'd still like a way to empty beakers without making the floor dirty.

**Comment by Exxion** [comment]
Dumping things down sink drains when?

**Comment by 9600bauds** [comment]
But sink is for water

**Comment by Exxion** [comment]
Sometimes things can do more than one thing

**Comment by 9600bauds** [comment]
How u make controls for doing both of the thing??

**Comment by Exxion** [comment]
Verb or Ctrl+click with container

**Comment by 9600bauds** [comment]
Added a dubious version of the dumping things down sink request, let me know if you can think of a better way to implement this

**Comment by Probe1** [comment]
>dumping the contents of a urinal
thanks but no thanks

**Comment by 9600bauds** [comment]
You are dumping INTO it, silly

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -368,10 +406,10 @@
 
 	//Note : Remember process() rechecks this, so the mix/max procs slowly increase/decrease body temperature
 	if(watertemp == "freezing cold")
-		C.bodytemperature = max(T0C - 10, C.bodytemperature - 0.5) //Down to -10�C - sorry.
+		C.bodytemperature = max(T0C - 10, C.bodytemperature - 0.5) //Down to -10°C - sorry.
```
Comment (9600bauds) [inline code/game/objects/structures/watercloset.dm]: fug

Comment (9600bauds) [inline code/game/objects/structures/watercloset.dm]: I can't get it to stop happening

**Comment by Probe1** [comment]
nice try you almost snuck poop back into the game good thing I was here.

**Comment by 9600bauds** [comment]
Probe my man do you shit into urinals?

**Comment by Probe1** [comment]
On a space station with clowns?

**Comment by Intigracy** [comment]
Change your addresses 2. of # to just say closes # since it's the last bullet point to address anyway

**Comment by ghost** [comment]
So, this makes it so you can't accidentally viraloose when clearing out vials?

Neat

**Comment by Exxion** [comment]
Well, you definitely still can, but it would require substantially more stupidity

**Comment by 9600bauds** [comment]
@Intigracy I actually already closed that issue because I'm that overconfident

**Comment by nuklearcellphoneg** [comment]
PLUMBING SYSTEM WHEN

**Comment by Exxion** [comment]
When
You
Meme
It

**Pull request merged**